01_Setup/README.md

```diff
@@ -24,10 +24,10 @@ $ sudo dnf install ansible python2-openshift
 It is beneficial to install the dependencies in a virtual environment to avoid
 installing unneeded files on your local machine. To do this, ensure you have
 `virtualenv` installed and run:
+
 ```
-$ mkdir -p ~/operatordev/<project>
-$ virtualenv ~/operatordev/<project>
-$ source ~/operatordev/foo/bin/activate
+$ virtualenv /tmp/operatordev
```

Comment:
This is because my preference is to not put stuff in my homedir. In tmp, if I totally forget about this after going through the training it will disappear after a reboot.

Reply:
I'd vote to not use /tmp by default. On the other side of the coin there are plenty of people who may want to finish the lab later, or they accidentally run out of battery mid-lab and their computer shuts down, etc. My assumption is that anyone who would like to "live dangerously" ;) will put it in /tmp without us needing to suggest it.